README.md

```diff
@@ -81,6 +81,14 @@ Profiling using eBPF incurs lower overhead & provides kernel stacks. This (curre
 **By default, gProfiler sends logs to Granulate Performance Studio** (when using `--upload-results`/`-u` flag)
 This behavior can be disabled by passing `--dont-send-logs` or the setting environment variable `GPROFILER_DONT_SEND_LOGS=1`.
 
+### Metrics and metadata collection
+By default, the gProfiler agent sends system metrics (CPU and RAM usage) and metadata to the Performance Studio.
+The metadata includes system metadata like the kernel version and CPU count, and cloud metadata like the type of the instance you are running on.
+You can disable metrics and metadata by using the following parameters:
+* Use `--disable-metrics-collection` to disable metrics collection
```

Comment:
Any reason you split it into 3 types?

Now we have:
* `-u` for upload in general
* logs
* metrics
* metadata
* arguments

kinda confusing.

Also, some are already included by others (e.g, args are logged so "logs" imply "arguments").

Let's narrow it? I think that simply arguments can be merged into "metadata".
"metrics" can remain separate since it really doesn't contain (and shouldn't contain) anything identifying - just raw numbers.